tests/storage/datavolume.go

```diff
@@ -1094,6 +1097,63 @@ var _ = SIGDescribe("DataVolume Integration", func() {
 			Entry("[test_id:5896]with thick provision false, fstrim will make the image smaller", addThickProvisionedFalseAnnotation, true),
 		)
 	})
+
+	Context("VirtualMachinePreference", func() {
```

Comment:
What about something like `With VirtualMachinePreference and PreferredStorageClassName` and then adjust the two cases to something like `should create PVC using PreferredStorageClassName when storage class not provided by VM` and `should always create PVC using VM defined storage class over PreferredStorageClassName`?